src/AppKit/Enums.cs

```diff
@@ -327,9 +327,11 @@ public enum NSControlTint : ulong {
 	
 	[Native]
 	public enum NSControlSize : ulong {
-		Regular, 
-		Small,
-		Mini
+		Regular = 0, 
+		Small = 1,
+		Mini = 2,
+		[Mac (10,16)]
+		Large = 3
```

Comment:
minor: add a `,` so any future addition won't make the last value part of the diff